app/src/main/java/ch/epfl/sdp/blindly/user/User.kt

```diff
@@ -215,22 +233,26 @@ class User private constructor(
          * @return a String containing the age of the User
          */
         @RequiresApi(Build.VERSION_CODES.O)
-        fun getUserAge(user: User?): String? {
-            val birthday = user?.birthday?.split('.')
+        fun getUserAge(user: User?): Int? {
+            val birthday = user?.birthday
             if (birthday != null) {
-                val age = Period.between(
-                    LocalDate.of(
-                        birthday[2].toInt(),
-                        birthday[1].toInt(),
-                        birthday[0].toInt()
-                    ),
-                    LocalDate.now()
-                ).years
-
-                return age.toString()
+                return getAgeFromBirthday(birthday)
             }
             return null
         }
+
+        @RequiresApi(Build.VERSION_CODES.O)
+        fun getAgeFromBirthday(birthday: String): Int {
```

Comment:
I might be wrong, but I think this function can be private.

Reply:
It unfortunately cannot, I didn't know where to put this function but it's used in the setUser in the UserHelper (to compute the age range) and it didn't feel right to put this function in the UserHelper (it's already long enough) :')